doc/user_guide/built_ins.rst

```diff
@@ -135,111 +145,115 @@ the code parser) the differences between the original algorithm code
 and the translated algorithm code are:
 
 * The generic calls to ``invoke`` have been replaced by specific
-  ``CALL invoke_xx``. The calls within the invoke are removed, as are
-  duplicate arguments and any literals leaving the three fields being
-  passed in;
-* A use statement is added for the each of the new ``CALL invoke_xx``
+  ``CALL invoke_<xx>``. The calls within the ``CALL invoke_jacobi_mass_lump``
+  are removed, as are duplicate arguments and any literals leaving the
+  five fields and one operator being passed in;
+* A ``use`` statement is added for the each of the new ``CALL invoke_<xx>``
   which will call the generated PSy layer code.
 
-The existence of a call to a Built-in has made no difference at this point:
+The existence of calls to Built-ins has made no difference at this point:
```

Comment:
We can use the shortcut of '::' at the end of this line and remove the '::' on the line below.

Reply:
Done.